### DISCUSSION TITLE
Help Creating Custom Ordinal Explorer Page

### DISCUSSION THREAD

**Discussion by jbrace02** [ordinals/ord#1954] [Q&A]
Overview of the project:
The goal of the project is to extract ordinal data from the OP_Return field and display ordinals/inscriptions on a website. 

Progress and current work flow:

- Fully Indexed Bitcoin Core

-MongoDB: This allows for efficient storage, querying, and analysis of the ordinal data obtained from the OP_Return fields.

High-level workflow:

 - Synchronize the Bitcoin Core node

- Extract OP_Return data: Query the Bitcoin Core node to retrieve the relevant transactions containing OP_Return fields. Extract the ordinal data from these fields using appropriate parsing methods, depending on the format or encoding scheme present.

- Store extracted data in MongoDB: Once the ordinal data is extracted from the OP_Return fields, store it in the MongoDB database. This enables efficient storage and organization of the data, as well as providing a convenient way to query and analyze it.

- Integration with the website: Develop a web application that connects to the MongoDB database to display the extracted ordinal data. This may involve implementing a front-end interface for users to interact with the data and a back-end server that handles communication with the database.

Current problems and challenges:

Identifying the best approach and libraries to extract and process ordinal data from the OP_Return field.
Integrating the extracted ordinal data into the website for display and analysis.
Next steps and questions for the ordinal community:

1.) Are there any best practices or recommended tools for extracting ordinal data from the OP_Return field?

2.) How should we handle different formats or encoding schemes that might be present in the OP_Return field when extracting ordinal data?

3.) Can you provide any guidance on integrating the extracted ordinal data into a website for display?

**Comment by nammaki** [comment]
I don't think it even uses OP_RETURN 🤔

**Reply by schlunsen** [reply]
Checkout https:///ordeets.com
